packages/webdriver/tests/utils.test.js

```diff
@@ -52,8 +52,13 @@ describe('utils', () => {
     })
 
     it('getPrototype', () => {
-        const proto = getPrototype()
-        expect(typeof proto).toBe('object')
-        expect(typeof proto.status.value).toBe('function')
+        const jsonWireProtocolPrototype = getPrototype()
+        expect(typeof jsonWireProtocolPrototype).toBe('object')
```

Comment:
The test will pass when a Object or an Array is passed.
I suggest using `expect(jsonWireProtocolPrototype instanceof Object).toBe(true)` instead